### PR TITLE
create calc interface

### PR DESCRIPTION
**Pull request by sameh080081**
this pull request for calculator button and result screen within calc interface.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      <button type="button">3</button>
+      <button type="button" className="orange">+</button>
+    </div>
+    <div>
+      <button type="button" className="zero">0</button>
+      <button type="button">.</button>
+      <button type="button" className="orange">=</button>
+    </div>
+  </div>
+);
```
Comment (Whoistolu) [inline src/components/Calculator.js]: - [x] [REQUIRED] Great implementation @sameh080081 , however, the project requirement states that this component should be class-based. Kindly implement this to get the project approved.

File: src/App.js [hunk 2]
```diff
@@ -1,27 +1,8 @@
...
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  return (<Calculator />);
 }
```
Comment (Whoistolu) [inline src/App.js:6]: - [x] [REQUIRED] Great implementation @sameh080081 , however, the project requirement states that this component should also be class-based. Kindly implement this to get the project approved.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @sameh080081 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 2 above)
Comment (sameh080081) [reply src/App.js:6]: Hello @Whoistolu, class based component causes eslint error

File: src/components/Calculator.js (hunk 1 above)
Comment (sameh080081) [reply src/components/Calculator.js]: @Whoistolu 
Component should be written as a pure function  react/prefer-stateless-function
i got this eslint error if i use class based component

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @sameh080081 ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions

- [x] You may kindly add the correct [linter files](https://github.com/microverseinc/linters-config/tree/master/react-redux).

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**